infra/Pos/Network/CLI.hs

```diff
@@ -141,51 +193,50 @@ monitorStaticConfig :: forall m. (
                     -> Peers NodeId    -- ^ Initial value
                     -> m T.StaticPeers
 monitorStaticConfig cfg@NetworkConfigOpts{..} origMetadata initPeers = do
-    events :: Chan (MonitorEvent m) <- liftIO $ newChan
-
-    let loop :: Peers NodeId
```

Comment:
~Useless~ Very subjective change, should be avoided. May lead to extra conflicts, make reviewers spend their time, etc. and doesn't make life much better.